Makefile

```
@@ -101,6 +105,11 @@ packer-fmt: ##@packer Rewrite configuration files to a canonical format and styl
 	packer-images \
 	bash packer-fmt.sh
 
+.PHONY: test
+test: ##@test Run CI tests on travis to all build steps
+	@echo "Starting CI tests..."
+	bash test.sh
```

So, maybe this approach is better and we don't need copy the test.sh ;)

```suggestion
	bash hack/test.sh
```

---

I've tried use this approach, but the script was execute the paths inside the `hack/` path. The make targets still running in `/`